apollo-router/Cargo.toml

```diff
@@ -72,6 +72,7 @@ base64 = "0.21.4"
 bloomfilter = "1.0.12"
 buildstructor = "0.5.4"
 bytes = "1.5.0"
+chrono = "0.4.31"
```

Comment:
In the new telemetry PR we introduced https://docs.rs/time, do you think you can use this one instead of chrono ? It's lighter. And if not could you add  `default-features = false` and only add features you need on chrono ?

Reply:
I'm using @Geal 's suggestion and using SystemTime.